### PR TITLE
14.0 mig account export csv

### PR DESCRIPTION
**Pull request by Kev-Roche**

### CODE REVIEW

File: account_export_csv/wizard/account_export_csv.py [hunk 1]
```diff
@@ -245,13 +242,14 @@ def action_manual_export_journal_entries(self):
...
                 base64.encode(file_data, base64_data)
                 base64_data.seek(0)
-                self.env.cr.execute(
-                    """
-                UPDATE account_csv_export
-                SET data = %s
-                WHERE id = %s""",
-                    (base64_data.read(), self.id),
-                )
+                self.write({"data": base64_data.read()})
```
Comment (StefanRijnhart) [inline account_export_csv/wizard/account_export_csv.py]: Did you check the docstring of this method about the memory usage? If you change it to ORM, maybe you can provide some benchmarks to show that its memory usage is no longer of a different order than using SQL?

**Review by chusamo — APPROVED** [review]
Tested on runbot LGTM

File: account_export_csv/wizard/account_export_csv.py [hunk 2]
```diff
@@ -0,0 +1,367 @@
...
+                base64.encode(file_data, base64_data)
+                base64_data.seek(0)
+                self.write({"data": base64_data.read()})
+                # self.env.cr.execute(
+                #     """
+                # UPDATE account_csv_export
+                # SET data = %s
+                # WHERE id = %s""",
+                #     (base64_data.read(), self.id),
+                # )
```
Comment (bealdav) [inline account_export_csv/wizard/account_export_csv.py]: Yes please @Kev-Roche could you reset the older commented code, as suggested by @StefanRijnhart. Thanks

Comment (Kev-Roche) [reply account_export_csv/wizard/account_export_csv.py]: @bealdav @StefanRijnhart I reset the older code, but as you can see, it brings an error, I don't understand why.

**Comment by rafaelbn** [comment]
Dear @Kev-Roche , thank you for this contribution. Please could you rebase and check comments of the reviews you got? Thank you 😄

**Pull request closed without merging**